### DISCUSSION TITLE
Github Vercel status says "Building" while it is actually already "deployed"

### DISCUSSION THREAD

**Discussion by zolotokrylin** [vercel/community#6847] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/nuon/website/P6aFUEmupwmSuDC6zR5Uyu92n8nF

### Steps to Reproduce

https://github.com/truflation/website/pull/112

<img width="941" alt="image" src="https://github.com/vercel/community/assets/1125014/4164de27-fafb-494f-8f88-a98c861fba28">

**Comment by swarnava** [comment]
I wonder if it's related to recent GitHub [Incident](https://www.githubstatus.com/incidents/bfkxf0w61z1n). Can you constantly reproduce this?

**Reply by zolotokrylin** [reply]
@swarnava , yes, since some time ago it is always like that. We don't get a GREEN (deployed) tick anymore

**Reply by zolotokrylin** [reply]
@swarnava we still have the same problem. Nothing has changed. Is there any advice on how to handle it?

**Reply by pawlean** [reply]
@zolotokrylin Have you got a recent example?

**Reply by zolotokrylin** [reply]
@pawlean it's exactly the same problem: once we open PR the status is stuck in "Building" and our CI check doesn't allow us to merge the PR because it considered the Preview branch not being deployed.
<img width="546" alt="image" src="https://github.com/vercel/community/assets/1125014/7ee30c85-3136-4af9-a48a-f7c47a080d2a">

**Reply by pawlean** [reply]
@zolotokrylin Have you tried turning Vercel off for PRs and switching it on again? You can get to this UI via `Settings` > `Git`.
![CleanShot 2024-07-05 at 15 42 19@2x](https://github.com/vercel/community/assets/17087373/72e8d624-dd5b-462b-ab81-fe372d3b1e0f)

**Reply by zolotokrylin** [reply]
I have done the action that was recommended. Let's see if it resolves the issue. Thank you.

**Reply by pawlean** [reply]
Let us know!

**Reply by zolotokrylin** [reply]
@pawlean after doing what you have suggested, the visual aspect of the problem is solved

<img width="841" alt="image" src="https://github.com/vercel/community/assets/1125014/4a2a4328-740c-400a-b731-18ee3ade9576">


but the CI is still stuck with that message "Missing successful active Preview deployment."
<img width="915" alt="image" src="https://github.com/vercel/community/assets/1125014/26694f48-f89f-4d44-85bd-43367e4751ea">

Here is how it looks in the settings of the github:

<img width="949" alt="image" src="https://github.com/vercel/community/assets/1125014/909978e7-bf9a-40bd-90e8-558e02b6abfe">

**Reply by pawlean** [reply]
@zolotokrylin And does the preview deployment link work?

**Reply by zolotokrylin** [reply]
@pawlean Yes, the link works.

**Reply by pawlean** [reply]
@zolotokrylin That is odd. Do you get this same behaviour in other projects in Vercel as well?

**Reply by zolotokrylin** [reply]
@pawlean Just in that one. Not sure if this is related, but that project is a private fork.

**Reply by pawlean** [reply, marked as answer]
Interesting - sorry for more testing, but can you try and adding the project again on Vercel and see if you see the same issue? If so, we can file a bug report.

**Reply by zolotokrylin** [reply]
@pawlean Do you mean this https://github.com/vercel/community/discussions/6847#discussioncomment-9968165 action?

**Reply by pawlean** [reply]
For some reason, the link you're sharing links back to this page. I don't know if you were linking to a specific comment?

But for clarity: I mean can you create a fresh new project on the Vercel Dashboard with this repo?

**Reply by zolotokrylin** [reply]
yes, to this comment 
<img width="975" alt="image" src="https://github.com/vercel/community/assets/1125014/54eb335e-d377-44c5-814d-b4015040d161">

> But for clarity: I mean can you create a fresh new project on the Vercel Dashboard with this repo?

ok, I will try.

[a comment by an automated account is not shown]
